library/consul

```diff
@@ -1,5 +1,11 @@
 Maintainers: Consul Team <consul@hashicorp.com> (@hashicorp/consul)
 
+Tags: 1.6.2, 1.6, latest
+Architectures: amd64, arm32v6, arm64v8, i386
+GitRepo: https://github.com/hashicorp/docker-consul.git
+GitCommit: 954cabfc1d2f76a92e41b1213c0422623f6596fa
+Directory: 0.X
+
 Tags: 1.6.1, 1.6, latest
```

Comment:
@mikemorris Entirely remove the section for `1.6.1` here. The old version will remain available for pull, but it will deliberately not show up in the list of available versions nor will it be updated again. That's what we've been doing for each major release.